packages/rate-limit-controller/src/RateLimitController.test.ts

```diff
@@ -45,6 +45,8 @@ function getRestrictedMessenger(
 ) {
   return controllerMessenger.getRestricted({
     name,
+    allowedActions: [],
+    allowedEvents: [],
   }) as RateLimitMessenger<RateLimitedApis>;
```

Comment:
Can this be removed now? I noticed you removed some type casts in other files.

Reply:
Good catch! I missed that one. Removed here: https://github.com/MetaMask/core/pull/4035/commits/9cdcbf1462806184138d6bfa090dfb88c3192a61